src/BaGetter.Core/BaGetter.Core.csproj

```diff
@@ -4,6 +4,7 @@
 
     <PackageTags>NuGet</PackageTags>
     <Description>The core libraries that power BaGetter.</Description>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
```

Comment:
I don't think this is always the right way. BaGetter.Core has a readme describing the source folder contents, which does not fit as package description on the gallery page. I'm not sure how to handle this though, since the readme here does fit a github page, describing its contents. Maybe an additional `PackageReadme.md` file for information to display on the gallery page?